libraries/ae-inventory-processor/src/main/java/org/metaeffekt/core/inventory/processor/patterns/ComponentPatternProducer.java

```diff
@@ -553,9 +556,19 @@ private ComponentPatternContributorRunner getAllContributors() {
         contributorRunnerBuilder.add(new JavaRuntimeComponentPatternContributor());
         contributorRunnerBuilder.add(new JettyComponentPatternContributor());
         contributorRunnerBuilder.add(new WebApplicationComponentPatternContributor());
-        contributorRunnerBuilder.add(new SystemBinaryComponentPatternContributor());
         contributorRunnerBuilder.add(new ProgressiveWebAppComponentPatternContributor());
         contributorRunnerBuilder.add(new ApkPackageContributor());
+        contributorRunnerBuilder.add(new AlpmPackageContributor());
+        contributorRunnerBuilder.add(new RpmPackageContributor());
+        contributorRunnerBuilder.add(new ConanComponentPatternContributor());
+        contributorRunnerBuilder.add(new GoLangComponentPatternContributor());
+        contributorRunnerBuilder.add(new JenkinsPluginsComponentPatternContributor());
+        contributorRunnerBuilder.add(new LinuxKernelArchiveContributor());
+        contributorRunnerBuilder.add(new LinuxKernelModulesContributor());
+        contributorRunnerBuilder.add(new NugetComponentPatternContributor());
+        contributorRunnerBuilder.add(new PubComponentPatternContributor());
+        contributorRunnerBuilder.add(new CocoapodsComponentPatternContributor());
+        contributorRunnerBuilder.add(new MixComponentPatternContributor());
 
```

Comment:
Check again, whether SystemBinaryCPC should still be used. Consider second pass to collect items not covered by other contribution pattern contributors. --> Separate PR.